sdk/python/lib/pulumi/output.py

```diff
@@ -536,6 +539,130 @@ def format(
             )
         return Output.from_input(format_string).apply(lambda str: str.format())
 
+    @staticmethod
+    def jsonDumps(
```

Comment:
Would it be more pythonic to name this `json_dumps`? cc: @kpitzen

Reply:
It would, too much switching between different languages 🤦